defs.bzl

```diff
@@ -4,33 +4,24 @@ Binary buildifier / buildozer setup
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
-
-_VERSION = "4.2.3"
-_BINARIES = [
-    # platform, arch, buildifier sha, buildozer sha
-    (
-        "darwin",
-        "amd64",
-        "954ec397089344b1564e45dc095e9331e121eb0f20e72032fcc8e94de78e5663",
-        "edcabae1d97bdc42559d7d1d65dfe7f8970db8d95d4bc9e7bf6656a9f2fb5592",
-    ),
-    (
-        "darwin",
-        "arm64",
-        "9434043897a3c3821fda87046918e5a6c4320d8352df700f62046744c4d168a3",
-        "f8d0994620dec1247328f13db1d434b6489dd007f8e9b961dbd9363bc6fe7071",
-    ),
-    (
-        "linux",
-        "amd64",
-        "a19126536bae9a3917a7fc4bdbbf0378371a1d1683ab2415857cf53bce9dee49",
-        "6b4177321b770fb788b618caa453d34561b8c05081ae8b27657e527c2a3b5d52",
-    ),
-]
+load("//:buildtools.bzl", "buildtools")
 
 def _buildifier_toolchain_setup_impl(repository_ctx):
     content = 'load("@buildifier_prebuilt//:toolchain.bzl", "declare_toolchain")'
 
+    # for platform, arch, _, _ in _BINARIES:
```

Comment:
@keith What do you think about defining a toolchain per tool? In other words, defining one for `buildifier` and one `buildozer`. This would allow the user to only download the tool that they are going to use.

Reply:
I'm actually wondering if we can drop the toolchain things, since it seems like bazel itself had the bug that lead me to toolchains, and they "fixed it" by replacing the labels with aliases, which apparently stops the unnecessary downloads. I want to verify that is stable behavior and then maybe we could even simplify this and remove toolchains entirely

Reply:
I was part of the way there. So, I ended up separating them. Let me know what you think.